CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # CHANGELOG
 
+## 1.0.0-beta.7
+
+### Features
+
+- renames Error Icon to AlertCircle which is what the figma calls it
+- puts the old Warning triangle icon back
```

Comment:
nit: I think these would be "breaking changes"...thoughts?

Reply:
Error is not yet used on the dashboard so it will not break anything other than Mocha's open PR

Reply:
@erin-doyle any thoughts on these? I'm still not used to the changelog and what's what

Reply:
The following suggestions are kind of nit-picky but I'm just trying to put myself in the shoes of someone not on our team or not familiar with this work coming in and trying to understand what changed and why.  But here's some tweaks I think might help:

1. Maybe rephrase "which is what the figma calls it" to "which is how its named in the design system docs".  People outside of our team may not know what is meant by "the figma"
2. "puts the old Warning triangle icon back" could be elaborated on.  Maybe: "To reduce duplication updated the Warning icon to be a triangle instead of the  'i' already being used by the Info icon"
3. Maybe instead of using the word "fixes" below, if that really isn't a "bug fix" you could say "Updated".

Reply:
amazing, thank you!